api/src/utils/helperFunctions.ts

```diff
@@ -0,0 +1,3 @@
+export const wait = (milliseconds: number) => {
```

Comment:
Helper function, that enables a delay in the scrapping process. Useful when we don't know the element or request that must be completed before proceeding to a next step